<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>793</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1155</width>
    <height>793</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QUSPIN</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/icon/Quspin.ico</normaloff>
    <normalon>:/new/icon/Quspin.ico</normalon>:/new/icon/Quspin.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>112</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>112</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>336</x>
         <y>9</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Sensitive Axis&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonZ">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>46</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonY">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>78</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonDual">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>46</y>
         <width>48</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>Dual</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton033X">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>46</y>
         <width>48</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>0.33X</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton1X">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>46</y>
         <width>48</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>1X</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton3X">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>46</y>
         <width>48</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string>3X</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>9</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Analog Output Gain&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoadAll">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>46</y>
         <width>61</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load All</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonLoad">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>78</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSelectComPort">
       <property name="geometry">
        <rect>
         <x>662</x>
         <y>78</y>
         <width>150</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditLoadAll">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>46</y>
         <width>50</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditLoad">
       <property name="geometry">
        <rect>
         <x>820</x>
         <y>80</y>
         <width>50</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>660</x>
         <y>9</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Sensor Commands&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReboot">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>46</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reboot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonRescan">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rescan</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>10</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Restart Commands&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>322</width>
         <height>102</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Startup Commands&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonAutoStart">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Auto Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFieldZero">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Field Zero</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonCalibrate">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string>Calibrate</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>45</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonCPlot">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>1010</x>
         <y>80</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Charts</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonReboot_3">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>46</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Reboot</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAStart">
       <property name="geometry">
        <rect>
         <x>669</x>
         <y>45</y>
         <width>48</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>AStart</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Sensors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Logbook</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="textEditLogBook">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:7.8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QPushButton" name="pushButtonDateTime">
           <property name="geometry">
            <rect>
             <x>15</x>
             <y>370</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>DataTime</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButtonSave">
           <property name="geometry">
            <rect>
             <x>15</x>
             <y>410</y>
             <width>93</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>68</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>81</height>
          </size>
         </property>
         <property name="title">
          <string>File</string>
         </property>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>131</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Name of experiment:</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Name of Subject:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditExperiment">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>15</y>
            <width>311</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditObject">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>39</y>
            <width>311</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="labelDate">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>20</y>
            <width>111</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxSaveFile">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>40</y>
            <width>82</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Save file</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>68</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>81</height>
          </size>
         </property>
         <property name="title">
          <string>Experiment</string>
         </property>
         <widget class="QProgressBar" name="progressBar">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>40</y>
            <width>118</width>
            <height>23</height>
           </rect>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButtonStartStop">
          <property name="geometry">
           <rect>
            <x>370</x>
            <y>13</y>
            <width>121</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditDuration">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>89</x>
            <y>18</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEditPeriod">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>42</y>
            <width>41</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>19</y>
            <width>55</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Duration</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_8">
          <property name="geometry">
           <rect>
            <x>16</x>
            <y>44</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sample Rate</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_9">
          <property name="geometry">
           <rect>
            <x>143</x>
            <y>20</y>
            <width>16</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>s</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>145</x>
            <y>42</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Hz</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelProgress">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>17</y>
            <width>111</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBoxInfinite">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>18</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Infinite</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1155</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSetting"/>
    <addaction name="actionCharts"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionCharts">
   <property name="text">
    <string>Charts</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
